docs/src/pages/[platform]/components/heading/index.page.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Heading
-description: A Heading primitive is used for semantic section heading text representing six levels.
+description: Headings render HTML Section Heading elements <h1>–<h6>.
```

Comment:
nit: maybe just HTML Heading？

Reply:
+1 Can you adapt the previous version calling out the semantic meaning to the new format?
```
Heading renders semantic section heading text representing six levels
```